packages/bot-api/src/main/Bot.ts

```diff
@@ -99,15 +99,12 @@ export class Bot {
     this.account.on(PayloadBundleType.ASSET_ABORT, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.ASSET_IMAGE, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.ASSET_META, this.handlePayload.bind(this));
-    this.account.on(PayloadBundleType.AVAILABILITY, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.CALL, this.handlePayload.bind(this));
-    this.account.on(PayloadBundleType.CLEARED, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.CLIENT_ACTION, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.CONFIRMATION, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.CONNECTION_REQUEST, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.CONVERSATION_CLEAR, this.handlePayload.bind(this));
     this.account.on(PayloadBundleType.CONVERSATION_RENAME, this.handlePayload.bind(this));
-    this.account.on(PayloadBundleType.LAST_READ_UPDATE, this.handlePayload.bind(this));
```

Comment:
These were never emitted so I did not type them and removed them from the BotAPI.